src/redux/categories/CategoriesSlice.js

```diff
@@ -0,0 +1,20 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = [];
```

Comment:
- [ ] Hello! 😊 As pointed out by the previous reviewer, the initialState should have a state value for categories that contains an empty array. 📚 This means that when the initial state is set up, the categories property should start with no items in its array, creating a clean and empty slate for your application's category-related data. 📖✨

```sh
// ./src/redux/categories/categoriesSlice.js

import { createSlice } from "@reduxjs/toolkit"

export const categoriesSlice = createSlice({
  name: "categories",
  initialState: {
    categories: [],
  },
  reducers: {
    checkStatus: (state, action) => {
      state.categories =
        action.payload === "Under construction"
          ? "Under construction"
          : state.categories
    },
  },
})

export const { checkStatus } = categoriesSlice.actions

export default categoriesSlice.reducer

```